recipes/simdna/meta.yaml

```diff
@@ -0,0 +1,72 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "simdna" %}
+{% set version = "0.0.0.deepchem" %}
+{% set git_tag = "197da84" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#  `openssl sha256 <file name>`.
+# You may need the openssl package, available on conda-forge
+#  `conda install openssl -c conda-forge``
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/deepchem/simdna.git
```

Comment:
I'm wary of pulling from a different repo than what appears to be the official repo (https://github.com/kundajelab/simdna), although I'm not sure what the relationship is between the two. We don't want to cause confusion for users looking of something and getting another. 

Also, please see:

https://conda-forge.org/docs/meta.html#build-from-tarballs-not-repos